src/libpst.c

```diff
@@ -807,31 +823,32 @@ int pst_load_extended_attributes(pst_file *pf) {
 
 
 #define ITEM_COUNT_OFFSET32        0x1f0    // count byte
+#define MAX_COUNT_OFFSET32         0x1f1
+#define ENTRY_SIZE_OFFSET32        0x1f2
 #define LEVEL_INDICATOR_OFFSET32   0x1f3    // node or leaf
 #define BACKLINK_OFFSET32          0x1f8    // backlink u1 value
-#define ITEM_SIZE32                12
```

Comment:
These values (expected entry size, max entry counts) are included in the page data, so I decided to just read them there.  They're almost always the same value (and the max count should just be `page_entry_space // entry_size`), but the [spec](https://msdn.microsoft.com/en-us/library/ff385991(v=office.12).aspx) indicates that the specified entry size must be used (the entry size can be larger for alignment purposes, etc).